### PR TITLE
DP-22403: Make existing paragraph types on org page available for sections

### PR DESCRIPTION
**Pull request by weekbeforenext**
**Description:**
Added Featured Message, Featured Item Mosaic and Organization Grid to the Organization Section paragraph type.

**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-22403

**To Test:**
- Login and go to /node/add/org_page.
- Click on the "Content" tab.
- Expand the "Section Content" field.
- [ ] Verify there are buttons for: Add Featured Mosaic, Add Featured Message, Add Organization Grid.
- Click on "Add Featured Mosaic".
- [ ] Verify there is no heading field in the form.
- Click on the "Add Featured Message".
- [ ] Verify there is no heading field in the form.

**Screenshots/GIFs:**

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by weekbeforenext** [comment]
This PR has the same behat failures as the branch it is merging into.

File: conf/drupal/config/core.entity_form_display.paragraph.featured_message.default.yml [hunk 1]
```diff
@@ -35,17 +35,13 @@ content:
...
+    type: string_textfield
     weight: 0
+    region: content
     settings:
       size: 60
       placeholder: ''
-    third_party_settings:
-      maxlength:
-        maxlength_js: 60
-        maxlength_js_label: 'Content limited to @limit characters, remaining: <strong>@remaining</strong>'
```
Comment (arthurbaghdas) [inline conf/drupal/config/core.entity_form_display.paragraph.featured_message.default.yml:45]: Do we want to lose this functionality?

**Comment by arthurbaghdas** [comment]
There is a validation that needs to be refactored, I think. 
See docroot/modules/custom/mass_validation/mass_validation.module:1209

File: conf/drupal/config/core.entity_form_display.paragraph.featured_message.default.yml (hunk 1 above)
Comment (weekbeforenext) [reply conf/drupal/config/core.entity_form_display.paragraph.featured_message.default.yml:45]: Good catch. I'll reinstate.

**Pull request merged**